recipes/netcdf-cxx-legacy/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    export DYLD_LIBRARY_PATH=${PREFIX}/lib
```

Comment:
you may not need this if you get the rpaths from the the LDFLAGS

Reply:
Yep, I'm planning to take this out.  I thought it might fix the OSX issue but no dice.